sanity/keylime-secure_mount/test.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+# vim: dict+=/usr/share/beakerlib/dictionary.vim cpt=.,w,b,u,t,i,k
+. /usr/share/beakerlib/beakerlib.sh || exit 1
+
+SECURE_DIR="/var/lib/keylime/secure"
+KEYLIME_UNIT_FILE="/etc/systemd/system/keylime_agent.service"
+if [ ! -f  "$KEYLIME_UNIT_FILE" ]; then
+    KEYLIME_UNIT_FILE="/usr/lib/systemd/system/keylime_agent.service"
+fi
+
+rlJournalStart
+
+    rlPhaseStartSetup "Do the keylime setup"
+        rlRun 'rlImport "./test-helpers"' || rlDie "cannot import keylime-tests/test-helpers library"
+        rlAssertRpm keylime
+
+        # update /etc/keylime.conf
+        limeBackupConfig
+        rlFileBackup $KEYLIME_UNIT_FILE
+        rlRun "limeUpdateConf logger_root level DEBUG"
+        rlRun "limeUpdateConf logger_keylime level DEBUG"
+        rlRun "limeUpdateConf handler_consoleHandler level DEBUG"
+        # tenant
+        rlRun "limeUpdateConf tenant require_ek_cert False"
+        # if TPM emulator is present
+        if limeTPMEmulated; then
+            # start tpm emulator
+            rlRun "limeStartTPMEmulator"
+            rlRun "limeWaitForTPMEmulator"
+            rlRun "limeCondStartAbrmd"
+            # start ima emulator
+            rlRun "limeInstallIMAConfig"
+            rlRun "limeStartIMAEmulator"
+        fi
+        sleep 5
+        rlRun "limeStartVerifier"
+        rlRun "limeWaitForVerifier"
+        rlRun "limeStartRegistrar"
+        rlRun "limeWaitForRegistrar"
+    rlPhaseEnd
+
+    rlPhaseStartTest "Verify that agent creates mount point when not present and mounts tmpfs."
+        if [[ $(findmnt -M "$SECURE_DIR") ]]; then
+            rlRun "umount $SECURE_DIR"
+        fi
+        rlRun "rm -rf $SECURE_DIR"
+        sed -i '/var-lib-keylime-secure.mount/d' $KEYLIME_UNIT_FILE
+        rlRun "systemctl daemon-reload"
+        rlRun "limeStartAgent"
+        sleep 3
+        rlAssertGrep "INFO  keylime_agent::secure_mount > Directory \"$SECURE_DIR\" created" $(limeAgentLogfile)
+        rlRun "findmnt -M \"$SECURE_DIR\" -t tmpfs"
```

Comment:
Would be nice at least to add a comment like "verify that the mount point has been created and tmpfs mounted" prior this command (or as a rlRun 3d argument). Just to point to this verification step.